src/nav/infrastructure.yml

```diff
@@ -168,6 +168,8 @@ pages:
             path: /docs/infrastructure/host-integrations/host-integrations-list/envoy-integration
           - title: F5 integration
             path: /install/f5
+          - title: Github Integration
+            path: /docs/infrastructure/host-integrations/host-integrations-list/
```

Comment:
I am unable to open this page by clicking the corresponding entry in the left nav. Not sure why though. I notice there's a forward slash at the end of the path. Can we please delete it and check if that does the trick?

Reply:
Can you open now? Please check and let me know.